### DISCUSSION TITLE
Material law for simple isotropic elastic-plastic?

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#1849] [Q&A]
Hi everyone?

I have a question regarding the material law for simple isotropic elastic-plastic. I had a hard time to find a suitable material law in Radioss to simulate simple elastic-plastic material. Could you please let me know which law if the best for this type of material? for example, in abaqus I can use keywords as below and what the similar one in Radioss? Thanks for your time

*MATERIAL,NAME=DEFORM
*DENSITY
7850.,
*ELASTIC
200.E9,.3
*PLASTIC
250.E6,0.
700.E7,1.

Regards
Lee

**Comment by MarianBulla** [comment]
Hi Lee,
thanks for this questions. I think it is quite interesting for many users, who are currently using ABAQUS. So it is an imprtant question.

This is one of the most material laws, used to model isotropic elasto-plastic materials, like metals.

In OpenRadioss is has an exact equivalent in the LAW36 (or /MAT/PLAS_TAB/ ) just review in the manual for materials, LAW36:

/MAT/PLAS_TAB/1
example steel
#              RHO_I
               7850.                   0
#                  E                  Nu           Eps_p_max               Eps_t               Eps_m
              200e9                 .3                   0                   0                   0
#  N_funct  F_smooth              C_hard               F_cut               Eps_f                  VP
         1         0                   0                   0                   0                   0
#  fct_IDp              Fscale   Fct_IDE                EInf                  CE
         0                   0         0                   0                   0
# func_ID1  func_ID2  func_ID3  func_ID4  func_ID5
         1         
#           Fscale_1            Fscale_2            Fscale_3            Fscale_4            Fscale_5
                  1.0    
#          Eps_dot_1           Eps_dot_2           Eps_dot_3           Eps_dot_4           Eps_dot_5
                   0          
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
/FUNCT/1
this line is only a comment for tabulated function
#                  X                   Y
                   0                250e6                                                            
              1.0               700e7                                                           
                                                      

Be aware, in OpenRadioss you have to shitch the columns of the hardening curve (first column is the plastic strain and 2nd column is the true-stress).

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Thanks Marian. That is extremely helpful.  I have another question regarding the materials laws. what if I am given the hardening curve in /MAT/PLAS_JOHNS/ and specify iflag=1 as attached? is this going to preform the same or exact equivalent to LAW36 as you given above? Thanks for your time again

![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/68a10f05-029f-43be-824e-cfc03f36c64a)

**Comment by MarianBulla** [comment]
Hi Lee,
LAW2 (which is the Johnson-Cook material law), has an equation working behind, wheras LAW36 uses the tabulated function.
For the equation, you need 3 parameters, A, B and n. 
If I_flag is equal to zero, you have to input these 3 parameters (exactly the same as in other solvers, like Abaqus).
But if you set the I_flag to one, OpenRadioss requests the engineering/physical values and is performing the paramteridentification internally.
Therefore, you only need to research the physical values of your materials:
Yield Stress, UTS and strain at UTS. You have a good chance to find them somewhere at the sites of the manufacturers. Easier than finding the entire curve. 
But if you already have the curve, then go with LAW36.
Otherwise, use LAW2 with I_flag = 1:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/5f57eb98-c606-4385-bc5b-2531d903cdb4)

The results are quite good for metals. But they are not the same, because in LAW36 you use a curve from a real test, where in LAW2 you use only a model. If you adjust your curve for LAW36 based on the values of the Johnson-Cook equation, then yes, you will get exactly the same results.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Thanks for your comprehensive explanations.  That's helpful definitely. Thanks again

**Comment by MarianBulla** [comment]
Hi Lee,

I'm very happy beeing able to help you :-)

Thanks and best regards,
Marian
